src/network/network.cpp

```diff
@@ -713,7 +714,7 @@ void NetworkClientConnectGame(NetworkAddress address, CompanyID join_as, const c
 static void NetworkInitGameInfo()
 {
 	if (StrEmpty(_settings_client.network.server_name)) {
-		seprintf(_settings_client.network.server_name, lastof(_settings_client.network.server_name), "Unnamed Server");
+		strecpy(_settings_client.network.server_name, "Unnamed Server", lastof(_settings_client.network.server_name));
```

Comment:
why this change?

Reply:
Because, rightfully, an earlier review said we shouldn't be using `printf` if we are doing literal strings. And I happened to know another place that was doing it wrong. Call it a freebie.